src/Modal/marketing-modal.mdx

```diff
@@ -13,7 +13,15 @@ notes: |
 
 ---
 
-The composition of `ModalDialog` for marketing style scenarios. This component passes through all of its props to an underlying `ModalDialog` component providing some limited customization. If you have unique needs, use the `ModalDialog` compound component family directly.
+Marketing modal is similar to the alert modal provides friendly messaging to our users to encourage greater engagement than the simpler confirmation dialogs. They should enable platform understanding, highlight opportunities and support workflow engagement.
```

Comment:
Lol, even though this is straight from Figma, it reads a bit weird due IMO. Maybe:

```
``MarketingModal`` is similar to the alert modal, providing friendly messaging to our users to encourage greater engagement than the simpler confirmation dialogs. They should enable platform understanding, highlight opportunities, and support workflow engagement.

The ``MarketingModal`` is a preconfigured `ModalDialog` that accepts an image and center aligns buttons.
```

Reply:
Yeah, it is better, updating to this variant